library/modules/bigip_gtm_wide_ip.py

```diff
@@ -119,6 +119,42 @@
         module.
     type: str
     version_added: 2.8
+  persistence:
+    description:
+      - When C(enabled), ensures that when a local DNS makes repetitive requests on
+        behalf of a client, the system reconnects the client to the same resource
+        as previous requests.
+      - When C(disabled), ensures repetitive requests do not reconnect the client
+        to the same resource.
+    type: str
+    choices:
+      - disabled
```

Comment:
this should be boolean, change it to boolean, look at other modules how to do this

Reply:
changed. I was following the module's current pattern.